build.sbt

```diff
@@ -303,6 +303,23 @@ lazy val dagShared = (project in file("modules/dag-shared"))
     )
   )
 
+lazy val rosetta = (project in file("modules/rosetta"))
+  .enablePlugins(JavaAppPackaging)
```

Comment:
are you going to build this module as a separate jar file?

Reply:
No. This has been removed.